### ISSUE TITLE
[web][edge] unskip Edge semantics tests

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#50754]
Bunch of tests are skipped for semantics in Edge in the following files. We should either skip them or change them to work in in Edge . (note: there is a chance that the browser has a bug for the tests methods use case)

semantics_test.dart
semantics_helper_test.dart

**Comment by nturgut** [comment]
There is test state leaking in this file, so if some tests fail others also start failing.

Test fails due to an error in the matcher:

 Expected: '<sem style="filter: opacity(0%); color: rgba(0, 0, 0, 0)"><sem-c><sem aria-label="Hello"><sem-v>Hello</sem-v></sem></sem-c></sem>'
    Actual: '<sem style="color: rgba(0, 0, 0, 0); filter: opacity(0%)"><sem-c><sem aria-label="Hello"><sem-v>Hello</sem-v></sem></sem-c></sem>'
     Which: is different.
            Expected: ... em style="filter: op ...
              Actual: ... em style="color: rgb ...

Which means the css style order on DOM is different and the matcher fails the tests.

/cc @ferhatb

**Comment by yjbanov** [comment]
Edge is now the same as Chromium. We don't need special handling any more.
